packages/forma-36-react-components/src/components/Flex/Flex.tsx

```diff
@@ -172,15 +173,17 @@ export const Flex = ({
   });
 
   return (
-    <Container
+    <Primitive
       {...otherProps}
-      style={{ ...inlineStyle, ...marginResult, ...paddingsResult, ...style }}
+      as={as}
       className={classNames}
-      data-test-id={testId}
+      ref={forwardedRef}
+      style={{ ...inlineStyle, ...marginResult, ...paddingsResult, ...style }}
+      testId={testId}
     >
       {children}
-    </Container>
+    </Primitive>
   );
-};
+}) as Polymorphic.ForwardRefComponent<typeof DEFAULT_TAG, FlexProps>;
```

Comment:
I think there should be a way to make it automatically inferred without explicit typings.